packages/mobile/src/screens/settings-screen/AccountVerificationScreen.tsx

```diff
@@ -157,15 +157,16 @@ export const AccountVerificationScreen = () => {
       const handle = type === 'twitter' ? profile.screen_name : profile.username
       const verified =
         type === 'twitter' ? profile.verified : profile.is_verified
-      dispatch(signOnActions.validateHandle(handle, verified))
+
+      dispatch(signOnActions.validateHandle(accountHandle ?? handle, verified))
```

Comment:
is it safe to fall back to handle?

Reply:
I don't think that we should be able to get here without the user being logged it so I don't think we would ever fallback, but it seemed safe enough bc that's what we have been using until now